pubspec.yaml

```diff
@@ -1,15 +1,14 @@
 name: dart_code_viewer
 description: A package to view Dart Code in your Flutter application.
-version: 0.0.1
+version: 0.0.2
 homepage: https://github.com/JoseAlba/dart-code-viewer/
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0.0.dev <3.0.0"
```

Comment:
Maybe using a more tolerant constraint such as `">=2.10.0-0 <3.0.0"`?

Reply:
It doesn't let me because 2.10 doesn't have support for nndb